catalogs/catalog-jdbc-common/src/test/java/com/datastrato/gravitino/catalog/jdbc/test/TestJdbcAbstractIT.java

```diff
@@ -2,7 +2,7 @@
  * Copyright 2023 Datastrato Pvt Ltd.
  * This software is licensed under the Apache License version 2.
  */
-package com.datastrato.gravitino.integration.test.catalog.jdbc;
+package com.datastrato.gravitino.catalog.jdbc.test;
```

Comment:
Please unify all the package name to "xxx.integration.test"

Reply:
This seems not resolved.

Reply:
It's under the JDBC-common module, so I ingore it.

Reply:
done